Today/Controllers/ReminderDetailViewController/ReminderDetailDataSource.swift

```diff
@@ -72,7 +71,7 @@ class ReminderDetailDataSource: NSObject {
                     return nil
                 }
                 if Locale.current.calendar.isDateInToday(date) {
-                    return NSLocalizedString("Today", comment: "Today for date description")
+                    return NSLocalizedString(Strings.todayString, comment: Strings.todayForDateDescriptionString)
                 }
                 return Self.dateFormatter.string(from: date)
```

Comment:
Normally we put these time formatters in Date extension so we can use it anywhere in the app as there would be quite few timeformatters in the app.